tests/unit/test_utils.py

```diff
@@ -72,7 +72,7 @@ def test_bad_get_date_delta_type_error():
     """Text exception raised for utils.get_date_delta"""
     with raises(TypeError, match="foo"):
         utils.get_date_delta(datetime.datetime(2023, 4, 20, 13, 8, 30),
-        'foo')
+                             'foo')
```

Comment:
repeat Sean's nit about new lines.

Reply:
Not sure I understand the comment here?

This is a minimal change addressing the existing flake8 complaint that I saw when I ran it against the tests area.

Reply:
I changed it to the more fully expanded way of writing this as part of my cleanup & updates to the commit message.